doc/varnish-configuration.rst

```diff
@@ -213,5 +213,6 @@ Configure your Varnish to set a debug header that shows whether a cache hit or m
         :language: varnish3
         :linenos:
 
-.. _`default VCL`: https://www.varnish-cache.org/trac/browser/bin/varnishd/default.vcl?rev=3.0#L63
+.. _`builtin VCL`: https://www.varnish-cache.org/trac/browser/bin/varnishd/builtin.vcl?rev=4.0
+.. _`default VCL`: https://www.varnish-cache.org/trac/browser/bin/varnishd/default.vcl?rev=3.0
```

Comment:
i suggest removing the line anchor - things might move and then it becomes very confusing.